### DISCUSSION TITLE
Deduct overspent budget from next period

### DISCUSSION THREAD

**Discussion by ThxAndBye** [firefly-iii/firefly-iii#7405] [Firefly III] [answered]
I've set a monthly budget that is set to add an amount for every period.  

I've noticed that when the budget is overspent that the overspent amount isn't deducted from the next period but rather the next period starts with he full budget again.  

Is there a setting for this? I don't want the overspent amount to vanish in the next cycle.

**Comment by JC5** [comment]
You can try the rollover setting? It adds X amount to your budget each period, minus what you spent of course.

**Reply by ThxAndBye** [reply]
I've set it to "Add an amount every period" and the overspend isn't deducted from the next period.   

In the [documentation ](https://docs.firefly-iii.org/firefly-iii/concepts/budgets/) for "Rollover" it also mentioned that overspend budget isn't carried over to the next period so I was wondering if there is a possibility to change this behavior.

**Reply by JC5** [reply, marked as answer]
Ah I see what you mean. No, this isn't possible. I've written it down! https://github.com/firefly-iii/firefly-iii/issues/7414
